packages/legacy/core/App/constants.ts

```diff
@@ -1,5 +1,14 @@
+import { Dimensions, Platform } from 'react-native'
+import StaticSafeAreaInsets from 'react-native-static-safe-area-insets'
+
 import { PINValidationRules } from './types/security'
 
+export const SCREEN_WIDTH = Dimensions.get('window').width
+export const SCREEN_HEIGHT = Platform.select<number>({
```

Comment:
Why not `import { useWindowDimensions } from 'react-native'`? It should be reactive and update on-demand?